packages/framework/immutable-merge/src/Merge.test.ts

```diff
@@ -133,7 +133,7 @@ const singleWithChanges = {
 
 const _colorKey = 'color';
 
-const changeMeHandler = (...objs: (object | undefined)[]) => {
+const changeMeHandler = (...objs: (Record<string, unknown> | undefined)[]) => {
   // written always assuming only one entry
```

Comment:
This seems like an API change?

Reply:
This is a test, so I'm not too worried. If it compiles and the test passes, it should be fine.